packages/suite/src/support/messages.ts

```diff
@@ -3437,6 +3437,10 @@ export default defineMessages({
         id: 'TR_GAS_PRICE',
         defaultMessage: 'Gas price',
     },
+    TR_CURRENT_FEE_CUSTOM_FEES: {
+        id: 'TR_CURRENT_FEE_CUSTOM_FEES',
+        defaultMessage: 'Current fee:',
```

Comment:
This sounds like it is current chosen fee. Probably it would be better to call it `Current network fee:`